src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -803,6 +818,21 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
     </div>
   );
 
+  const renderNametagsToggle = () => (
+    <div className="toggle-room">
+      <h4 className="italic input-header">Display profile names on avatars</h4>
+      <label id="showNametags" className="switch">
+        <input
+          type="checkbox"
+          id="showNametags"
+          name="showNametags"
+          ref={register}
+        />
```

Comment:
We generally don't need to set the `id` prop on element in React, as it's barely ever used for anything. Technically an `id` must be unique within the page, so having it set on both the label and input here is an error I believe (though one that most browsers will silently ignore)

An exception to this is that using the 'label for' pattern of associating a label with its input requires the input to have an `id` set. Though another way of associating them is to nest the `input` element directly within the `label` element, as you have done here. So I don't believe either will need an `id` here.

Further Reading:

- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

Reply:
```suggestion
      <label className="switch">
        <input
          type="checkbox"
          name="showNametags"
          ref={register}
        />
```

Reply:
Thanks! Fixed